### PR TITLE
[rocksdb] RocksDB binding

### PR DESCRIPTION
**Pull request by hobinyoon**
From https://github.com/adamretter/YCSB/commit/07a3b0495343d79724ad0589a10d4fb438f58751

Update mvn jar versions.
Check rocksdb.dir.

### CODE REVIEW

File: bin/ycsb [hunk 1]
```diff
@@ -83,6 +83,7 @@ DATABASES = {
     "redis"        : "com.yahoo.ycsb.db.RedisClient",
     "rest"         : "com.yahoo.ycsb.webservice.rest.RestClient",
     "riak"         : "com.yahoo.ycsb.db.riak.RiakKVClient",
+    "rocksdb"      : "com.yahoo.ycsb.db.RocksDBClient",
```
Comment (risdenk) [inline bin/ycsb]: Can the package be changed to `com.yahoo.ycsb.db.rocksdb.RocksDBClient`?

File: rocksdb/pom.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved.
```
Comment (risdenk) [inline rocksdb/pom.xml]: This is a new file so only need 2016 not the 2012 -

File: rocksdb/src/main/java/com/yahoo/ycsb/db/RocksDBClient.java [hunk 3]
```diff
@@ -0,0 +1,321 @@
+package com.yahoo.ycsb.db;
```
Comment (risdenk) [inline rocksdb/src/main/java/com/yahoo/ycsb/db/RocksDBClient.java]: Missing license?

File: rocksdb/src/main/java/com/yahoo/ycsb/db/package-info.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2014, Yahoo!, Inc. All rights reserved.
```
Comment (risdenk) [inline rocksdb/src/main/java/com/yahoo/ycsb/db/package-info.java]: License header is wrong. use YCSB Contributors and 2016

File: rocksdb/src/main/java/com/yahoo/ycsb/db/package-info.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+/**
+ * The RocksDB Java binding for <a href="http://rocksdb.org/">RocksDB</a>.
+ */
+package com.yahoo.ycsb.db;
```
Comment (risdenk) [inline rocksdb/src/main/java/com/yahoo/ycsb/db/package-info.java]: Package change com.yahoo.yscb.db.rocksdb

File: rocksdb/src/main/java/com/yahoo/ycsb/db/RocksDBClient.java (hunk 3 above)
Comment (risdenk) [inline rocksdb/src/main/java/com/yahoo/ycsb/db/RocksDBClient.java]: package change com.yahoo.ycsb.db.rocksdb

**Comment by manolama** [comment]
Ping @hobinyoon Any updates on this one? Thanks!

**Comment by hobinyoon** [comment]
I haven't had a chance to work on this since then. I'll make an update in a couple days.

**Comment by busbey** [comment]
Replaced by #1052

**Pull request closed without merging**